<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WithdrawAsk</class>
 <widget class="QDialog" name="WithdrawAsk">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>133</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="messageLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>531</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="yesButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>82</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="noButton">
   <property name="geometry">
    <rect>
     <x>179</x>
     <y>82</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
